lmnet/lmnet/datasets/mscoco.py

```diff
@@ -225,10 +225,16 @@ def coco_category_id_to_lmnet_class_id(self, cat_id):
     @functools.lru_cache(maxsize=None)
     def _gt_boxes_from_image_id(self, image_id):
         """Return gt boxes list ([[x, y, w, h, class_id]]) of a image."""
+        classes = [class_name for class_name in self.classes if class_name is not "__background__"]
+        class_ids = self.coco.getCatIds(catNms=classes)
```

Comment:
how many ids do we have in `class_ids` ?
if this is only used in line 235, we should convert this to tuple.

Reply:
`class_ids` has `90`.

tuple OK, thanks.

Reply:
Oops I was wrong, we should use `set()` not `tuple()` for `class_ids`. `classes` can be a list.